addons/web/static/src/scss/search_view_mobile.scss

```diff
@@ -13,7 +13,7 @@
             height: 46px;
             margin-bottom: 10px;
             width: 100%;
-            background-color: $o-brand-primary;
+            background-color: $o-enterprise-color;
```

Comment:
You probably wanted to use `$o-brand-odoo`

Reply:
Changes done.